src/engine/TextOperationWithoutFilter.cpp

```diff
@@ -12,7 +12,7 @@ using std::string;
 
 // _____________________________________________________________________________
 size_t TextOperationWithoutFilter::getResultWidth() const {
-  size_t width = 2 + getNofVars();
+  size_t width = 3 + getNofVars();
```

Comment:
Is that correct for this PR, or does it belong into the other PR that adds a feature?

Reply:
Yes this shouldn't be in this PR